src/screens/NationDetailsContainer/NationDetailsScreen/index.js

```diff
@@ -23,30 +24,42 @@ class NationDetailsScreen extends Component {
       return <BackgroundImage/>;
     }
 
-    return (
-      <View style={styles.container}>
+
+   return (
+      <View style={styles.screenContainer}>
         <BackgroundImage/>
-        <FakeNavigationBar/>
-        <Text style={styles.title}>{nation.name}</Text>
-        {this._buildButtonsView()}
-        <ScrollView style={styles.scrollView}>
-          <MessageView style={[styles.messageView]}>
-            <Image source={AssetsImage.Placeholder.map} resizeMode='contain'/>
-          </MessageView>
-          <MessageView style={[styles.messageView]}>
-            <Image source={AssetsImage.Placeholder.achievements} resizeMode='contain'/>
-          </MessageView>
-          {this._buildAboutView(nation)}
-          {this._buildGovernmentalStructureView(nation)}
-          {this._buildFactsView(nation)}
-        </ScrollView>
+        <FakeNavigationBar navBarHidden='' />
+        <View style={styles.titleBarLarge}>
+          <Text style={styles.title}>{nation.name}</Text>
+        </View>
+        {this._buildTabBar()}
+        <View style={styles.bodyContainer}>
+          <ScrollView style={styles.scrollView}>
+{/* Fake Map panel */}
```

Comment:
Can you remove it if it's not used?

Reply:
I'd like to keep it for testing and demonstration and placement. We are working on these screens right now, so this will change/disappear very shortly.